code/script.js

```diff
@@ -0,0 +1,83 @@
+// DOM selectors
+const projects = document.getElementById('projects')
+const username = 'idanaslund'
+const profile = document.getElementById('profile')
+const formRepos = document.getElementById('formRepos')
+const sortRepos = document.getElementById('sortRepos')
+
+let API_URL = `https://api.github.com/users/${username}/repos`
+
+// Fetching profile info
+const addingProfile = () => {
+    fetch(`https://api.github.com/users/${username}`)   
+    .then((res) => res.json())
+    .then((profileInfo) => {
+        profile.innerHTML += `
+        <img src="${profileInfo.avatar_url}">
+        <a class="userlink" href="${profileInfo.html_url}" target="_blank">${profileInfo.login}</a>
+        <p>${profileInfo.bio}</p>
+        `
+    }) 
+}
+addingProfile()
+
+// The first fetch of repos, calling the function below all other functions
+
+const addingRepos = () => {fetch(API_URL)  
+    .then((res) => res.json())
+    .then((repos) => {
+       
+     // Filtering all repos I've forked and that starts with the word project
+        const forkedRepos = repos.filter(
+            repo => repo.fork && repo.name.startsWith('project-')
+            ) 
+    // For all filtered repos I put the following info on my page      
+        forkedRepos.forEach((repo) => {
+            let updated = new Date(repo.updated_at).toLocaleDateString()            //Turning date and time into date
+            let upperCase = `${repo.name[0].toUpperCase()}${repo.name.slice(1)}`    //Making first letter of string to uppercase
```

Comment:
Great with the comments and nice to know how you wrote the code to make the first letter uppercase (I'm taking that with me since I tried that too but I didn't get it to work) :)